app_spec/test/many-agent-tests.js

```diff
@@ -0,0 +1,28 @@
+module.exports = scenario => {
+
+
+scenario('can run a test with many agents', async (s, t, instanceMap) => {
+  const nodes = Object.values(instanceMap)
+  const anchor = nodes[0]
+
+  t.ok(anchor.agentAddress)
+
+  for (const [name, node] of Object.entries(nodes)) {
+    await node.call('simple', 'create_link', {
```

Comment:
Should this be
```suggestion
    await node.callSync('simple', 'create_link', {
```
or is the new Diorama call function sync already and callSync just an alias?

Reply:
@lucksus the waiting happens now through `s.consistent()`. The `callSync` in the current app spec test is actually syntactic sugar added by the `backwardsCompabilityMiddleware`, used to minimize the changes needed. The intention of this test is to have each create_link happen as quickly as possible, though, without waiting in between.